tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,35 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class Bugzilla_lite(Base):
```

Comment:
Nit: We follow PEP8 for the style guide. A class must written as `ClassName` instead of `Class_Name`. So here, we should have:
``` python
class BugzillaLite(Base):
```

https://www.python.org/dev/peps/pep-0008/#class-names